pointcloud/basic_octree.go

```diff
@@ -13,8 +17,10 @@ const (
 	leafNodeFilled
 	// This value allows for high level of granularity in the octree while still allowing for fast access times
 	// even on a pi.
-	maxRecursionDepth = 1000
-	nodeRegionOverlap = 0.000001
+	maxRecursionDepth   = 1000
+	nodeRegionOverlap   = 0.000001
+	confidenceThreshold = 60
```

Comment:
The `confidenceThreshold` and `buffer` should be passed in and not global variables

Reply:
See below comment on splitting off into motion_octree type